### DISCUSSION TITLE
Tranfert to another computer and other IP [RESOLVED]

### DISCUSSION THREAD

**Discussion by garfield50** [amidaware/tacticalrmm#693] [General]
Hi,

I use a personnal computer Tactical RMM traditional for testing Tatical. And i using it to more than 250 computers and servers. I want to transfert to a real server to another location IP.

How can i do without crash :D

thank you.

**Comment by dinger1986** [comment]
essentially you want to back it up and restore it to another location.

Will DNS be the same?

Discord would be best to get support on this I would suggest

**Reply by garfield50** [reply]
thank you, i can transfert DNS to another IP
 :)

I hope i dont need install again agent.

**Comment by dinger1986** [comment]
Follow these guides:

https://wh1te909.github.io/tacticalrmm/backup/ and https://wh1te909.github.io/tacticalrmm/restore/

**Reply by garfield50** [reply]
thank you :) i try it ;)

**Reply by garfield50** [reply]
i have update my last server to 0.8.3 and i backup with last backup script. After, i tranfert my backup to new server and i change my dns ip recorder to new IP and, i restore my backup with same username from old server. But, when i try to connect to agent it ask login an password to meshcentral. i reset meshcentral password with node node_modules........... it's working, but from 250 computers, just 16 working and other was disconnected and i cant connect to their. i try restore recover mesh agent but i didnt work.

are their any solution or i need reinstall all tactical agent on compouters and servers ?

Thank you.

**Reply by dinger1986** [reply]
I think you need to generate a mesh token https://meshcentral2.blogspot.com/2021/04/meshcentral-login-tokens-mobile.html and put that into tactical under global settings

**Reply by garfield50** [reply]
thanks, i try it :)

**Reply by garfield50** [reply]
not working always 234 agent disconnected. Tacticalrmm i login perfect, the same to meshcentral but agents from computers and servers was offlines

**Reply by dinger1986** [reply]
have they resolved the new ip for the dns? Also have you opened all the ports?

**Reply by garfield50** [reply]
80 443 and 4222 are open from my router and i dont use firewall at this moment maybe later.

i try ping my domain rmm mesh and api and the ip is good from one offline server

**Reply by dinger1986** [reply]
can you browse to rmm.yourdomain.com and get the correct one?

**Reply by garfield50** [reply]
yes, i'm going to login page

**Reply by dinger1986** [reply]
weird, can you restart the server?

**Reply by garfield50** [reply]
I restart computer and tactical server Many time

**Reply by dinger1986** [reply]
I meant the server that is an agent. 

You could also try netstat to check what outbound connections are being made on the agent.

**Reply by garfield50** [reply]
this morning, many agent was up, maybe DNS propagation was longer than usual. But yesterday i ping with domain and ip was good.

for me the probleme was close.

Just something, webgui report bad agent status (overdue for more than 80% status agent)

i m going in server maintenant and reload NATS configuration as read from documentation, but that dont work.

maybe waiting future update was reload correctly configuration

**Reply by dinger1986** [reply]
any showing overdue are probably waiting on DNS propagation

**Comment by garfield50** [comment]
Hello,

I tranfered my tactical server on new computer, all working perfect. Just something, when i connect to agent for taking control, it's always ask me mesh central authentification, can i remove this step ? 

thanks.

**Reply by dinger1986** [reply]
Yes, regenerate the login token and put that in tactical

Steps are something like:

Run `node /meshcentral/node_modules/meshcentral --logintokenkey` then from tactical web ui go to Settings > Global Settings > Mesh Central and change Mesh Token to the output of that command

**Reply by garfield50** [reply]
thank you so much, problem has been resolved
